front/src/common/redux/services/authenticationService.js

```diff
@@ -0,0 +1,19 @@
+// import axios from 'axios';
+export const loginUserService = (request) => {
```

Comment:
Call it something like `loginApi`. Service is not an appropriate name for function because usually service is an actual object

Reply:
done